Gemfile

```diff
@@ -1,7 +1,7 @@
 source 'https://rubygems.org'
 
 # Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
-gem 'rails', '4.0.1'
```

Comment:
I think that portion at the end is just a summary of the explicit dependencies specified in the Gemfile. The longer list at the beginning seems to actually outline all the gems being used in the app and does include a very specific Rails version (line 78 in the new file). Unfortunately, I can't find any detailed docs that would confirm this